old-translations/es/guides/data-fetching.md

```diff
@@ -36,7 +36,7 @@ Si quieres usar `fetch()` en un componente que no sea Astro, usa la biblioteca [
 ```tsx
 // Movies.tsx
 import fetch from 'node-fetch';
-import type { FunctionalComponent } from 'preact';
+import type { FunctionComponent } from 'preact';
```

Comment:
preact v10 doesn't seem to suggest using `FunctionalComponent ` https://preactjs.com/guide/v10/typescript/#function-components though it's still exported.